showcase/README.md

```diff
@@ -153,26 +200,32 @@ $ cryptid controller add -c controlled.yml <UI_DID>
 
 ## Gateway
 
+Here we will show how to issue a gateway token to a cryptid address from a dummy gatekeeper. This example will show
+how you cannot mont a NFT until a gateway token has been issued.
 
-Goal: intro to gateway tokens, permissioned dApps and the gateway CLI
+You can follow the steps below and follow along on YouTube: https://www.youtube.com/watch?v=4oSTRnvnmNM
 
-- Visit [http://candy.identity.com](http://candy.identity.com) - controlled cryptid account cannot yet mint NFT (no GT)
+Visit [http://candy.identity.com](http://candy.identity.com) and notice that you cannot yet mint an NFT.
 
 ```shell
-$ gateway add-gatekeeper -c devnet $(solana address)
-
+# Issue a gateway token to your cryptid address
 $ gateway issue -g $HOME/.config/solana/id.json -c devnet <CONTROLLED CRYPTID ADDRESS>
 ```
 
-- Visit [http://candy.identity.com](http://candy.identity.com) - Mint NFT
+Visit [http://candy.identity.com](http://candy.identity.com) again, and notice that you can now mint an NFT.
 
 ## Credentials & Gateway
 
-Goal: "Real-life" example of a gateway token issuance based on credentials.
+Here is an end-to-end showcase to show a "real life" example of a gateway token issuance based on credentials. This will
+show how you cannot mint and NFT until you've gone through a Civic KYC process, signed the credentials with cryptid
```

Comment:
Best to mention (as we do in the video) the relationship between Civic and identity.com